scripts/validate-manifest.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# validate-manifest.sh [path/to/manifest.json]
+#
+# Dependencies: jq tr sort uniq
+#
+# Description: Ensures consistency of versions specified in manifest.json.
+
+set -eou pipefail
+
+if ! command -v jq > /dev/null; then
+    echo "This script requires jq to be available."
+    exit 1
+fi
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
This script could be run outside project root (at least theoretically, getting the script dir and cd-ing there could be useful).